loleaflet/js/global.js

```diff
@@ -476,7 +476,7 @@
 					{
 						// Throttle.
 						that.curPollMs = Math.min(that.maxPollMs, that.curPollMs * that.throttleFactor) | 0;
-//						console.debug('No data for ' + timeSinceLastDataMs + ' ms -- throttling to ' + that.curPollMs + ' ms.');
+						//						console.debug('No data for ' + timeSinceLastDataMs + ' ms -- throttling to ' + that.curPollMs + ' ms.');
```

Comment:
This doesn't seem right. It will look pretty weird when the `//` part is removed while debugging.